libnavui-voice/build.gradle

```diff
@@ -41,8 +41,6 @@ dependencies {
     api project(":libnavui-base")
     implementation project(":libnavui-resources")
 
-    api dependenciesList.mapboxSdkServices
```

Comment:
This will potentially break downstream upgrades which is essentially a SEMVER 🤔 . Why is this being moved?

Reply:
`libnavui-voice` doesn't need this dependency anymore.

The modules that implicitly referenced `mapboxSdkServices` classes via `libnavui-voice` are: `libnavui-dropin` and `examples`.

Reply:
Yeah `examples` represents a customer, so this will be a breaking change if are using `mapboxSdkServices`. The customer will not be able to simply, upgrade a minor version, they will need to add a dependency if they are depending on `mapboxSdkServices`

Seems we may want to add `api dependenciesList.mapboxSdkServices` to `libnavigation-android` to keep it as part of the sdk bundle 🤔

Reply:
`libnavui-voice` can be used independently of any other module, so removing a dependency that has been made public might in some circumstances break compilation downstream.

I believe that after this PR we don't have any dependencies on the HTTP services from mapbox-java so we could only rely on the models packages and decrease the binary size of the whole SDK. We could potentially take that decision (and document it appropriately) but let's discuss it separately from this PR. For now, let's leave the public dependency as-is.